docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.9.4-slim
+FROM node:16.13.0
```

Comment:
Let's use `16.13.1-slim` instead of the fat image

Reply:
The slim version don't have CURL and the file wait.sh in this line `curl -q "$1" 1>/dev/null 2>/dev/null` fail

Reply:
Should be fine with the non slim version